contrib/platform/src/com/sun/jna/platform/mac/CoreFoundation.java

```diff
@@ -488,6 +488,9 @@ public static CFStringRef createCFString(String s) {
          */
         public String stringValue() {
             CFIndex length = INSTANCE.CFStringGetLength(this);
+            if (length.longValue() == 0) {
+                return "";
+            }
             CFIndex maxSize = INSTANCE.CFStringGetMaximumSizeForEncoding(length, kCFStringEncodingUTF8);
             if (maxSize.intValue() == kCFNotFound) {
                 return null;
```

Comment:
This should be raising an error, as `kCFNotFound` means that the number exceeds `LONG_MAX`.

Reply:
Probably so. My code, I can fix in another PR if @dyorgio doesn't want to clean up my mess here.

Reply:
Hi guys, I included exception on kCFNotFound result but not in CFStringGetCString.
Normally people do that when use this function (return null) or don't care about result.
Apple says that it can be caused by 2 things: conversion fails or the provided buffer is too small.
I think that use of UTF-8 is safe for the first one and CFStringGetMaximumSizeForEncoding also put some guarantees for second case.